### PR TITLE
More python modules

### PR DESCRIPTION
**Pull request by paulfantom**
@ktsaou while you were debugging `python.d.plugin` I have created new class "UrlService" which eases creating services which are getting data from some http requests (like phpfpm). To test it I have created new phpfpm (and nginx one, but this will be added later) module.

It doesn't use any 3rd party libraries and is compatible with python2 and python3.

### CODE REVIEW

File: conf.d/python.d.conf [hunk 1]
```diff
@@ -5,5 +5,6 @@ enabled: no
 
 # By default python.d.plugin enables all modules stored in python.d
 # Modules can be disabled with setting "module_name = no"
-example: no
-mysql: no
+example: yes
+mysql: yes
+phpfpm: yes
```
Comment (ktsaou) [inline conf.d/python.d.conf]: please set them to no, until we obsolete to bash ones

File: conf.d/python.d/phpfpm.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+update_every: 3
```
Comment (ktsaou) [inline conf.d/python.d/phpfpm.conf]: ideally, I would love to have `update_every` on every module unset, to let it inherit the `update_every` supplied by netdata

File: python.d/phpfpm.chart.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# Description: PHP-FPM netdata python.d plugin
+# Author: Pawel Krupa (paulfantom)
+
+from base import UrlService
+
+# default module values (can be overridden per job in `config`)
+update_every = 2
```
Comment (ktsaou) [inline python.d/phpfpm.chart.py]: same here

**Comment by paulfantom** [comment]
Here you go:)

**Comment by ktsaou** [comment]
You forgot python.d.conf

**Comment by paulfantom** [comment]
But we are disabling everything with `enabled = no`.

**Comment by ktsaou** [comment]
> But we are disabling everything with enabled: no.

ok. My bad.

**Pull request merged**